packages/tasklist-client-node-js/src/__test__/restclient.spec.ts

```diff
@@ -44,130 +41,99 @@ describe('TasklistRESTClient', () => {
-        expect(claimTask.id).toEqual(taskId)
-    })
+        it('will not allow a task to be claimed twice', async () => {
+            const tasklist = new TasklistRESTClient()
+            const tasks = await tasklist.getTasks({ state: TaskStateREST.CREATED })
+            const task = await tasklist.assignTask({ taskId: tasks[0].id, assignee: 'jwulf' })
+            expect(task).toBeTruthy()
+            let threw = false
+            try {
+                await tasklist.assignTask({ taskId: tasks[0].id, assignee: 'jwulf0', allowOverrideAssignment: false })
+            } catch (e) {
+                threw = true
+            }
+            expect(threw).toBe(true)
+        })
 
-    it('can complete a Task', async () => {
-        const tasklist = new TasklistRESTClient()
-        const tasks = await tasklist.getTasks({ state: TaskState.CREATED })
-        const taskid = tasks[0].id
-        expect(tasks.length).toBeGreaterThan(0)
-        const completeTask = await tasklist.completeTask(taskid, { outcome: 'approved', fruits: ['apple', 'orange'] })
-        expect(completeTask.id).toBe(taskid)
-        p = null as any
-    })
+        it('can unclaim task', async () => {
+            const tasklist = new TasklistRESTClient()
+            const tasks = await tasklist.getTasks({ state: TaskStateREST.CREATED })
+            const taskId = tasks[0].id
+            const task = await tasklist.assignTask({ taskId: taskId, assignee: 'jwulf', allowOverrideAssignment: false })
+            expect(task).toBeTruthy()
+            let threw = false
+            try {
+                await tasklist.assignTask({ taskId: taskId, assignee: 'jwulf0', allowOverrideAssignment: false })
+            } catch {
+                threw = true
+            }
+            expect(threw).toBe(true)
+            await tasklist.unassignTask(taskId)
+            const claimTask = await tasklist.assignTask({ taskId, assignee: 'jwulf0', allowOverrideAssignment: false })
+            expect(claimTask.id).toEqual(taskId)
+        })
 
-    it('can complete a task with variables', (done) => {
-        const tasklist = new TasklistRESTClient()
-        zbc.createProcessInstanceWithResult({
-            bpmnProcessId: 'TasklistTestProcess',
-            variables: {
-                name: 'Joe Bloggs',
-                age: 42,
-                interests: ['golf', 'frisbee'],
-            },
-        }).then((res) => {
-            console.log(res)
+        it('can complete a Task', async () => {
+            const tasklist = new TasklistRESTClient()
+            const tasks = await tasklist.getTasks({ state: TaskStateREST.CREATED })
+            const taskid = tasks[0].id
+            expect(tasks.length).toBeGreaterThan(0)
+            const completeTask = await tasklist.completeTask(taskid, { outcome: 'approved', fruits: ['apple', 'orange'] })
+            expect(completeTask.id).toBe(taskid)
+            expect(completeTask.taskState).toEqual(TaskStateREST.COMPLETED)
```

Comment:
The specs got misaligned in the diff, so it's hard to see, but this assertion is added. 

This test on the right replaces _two_ tests from the left.

The original tests on the left assert `can complete a task` and `can complete a task with variables`, but I don't see any difference in (a) how they're completing a task, or (b) how they're asserting that the task was completed. I removed the second one because it was more complex, and tightened up the assertions of the first test to confirm that the task _was_ in fact completed.

If I'm not understanding that original `can complete a task with variables` test, let me know and I'll revisit!